validation/dam_break_2d/plot_dam_break_results.jl

```diff
@@ -0,0 +1,132 @@
+include("../validation_util.jl")
+
+# Activate for interactive plot
+# using GLMakie
+using CairoMakie
+using CSV
+using DataFrames
+using JSON
+using Glob
+using Printf
+using TrixiParticles
+
+# Initial width of the fluid
+H = 0.6
+W = 2 * H
+
+normalization_factor_time = sqrt(9.81 / H)
+normalization_factor_pressure = 1000 * 9.81 * H
+
+edac_reference_files = glob("validation_reference_edac*.json",
+                            "validation/dam_break_2d/")
+edac_sim_files = glob("validation_result_dam_break_edac*.json",
+                      "out/")
+
+merged_files = vcat(edac_reference_files, edac_sim_files)
+edac_files = sort(merged_files, by=extract_number_from_filename)
+
+wcsph_reference_files = glob("validation_reference_wcsph*.json",
+                             "validation/dam_break_2d/")
```

Comment:
These also need to use `joinpath` and `validation_dir`. Could that be the reason why CI is failing?